MdePkg/Include/IndustryStandard/PciExpress21.h

```diff
@@ -228,20 +236,21 @@ typedef union {
 
 typedef union {
   struct {
-    UINT16    SystemErrorOnCorrectableError : 1;
-    UINT16    SystemErrorOnNonFatalError    : 1;
-    UINT16    SystemErrorOnFatalError       : 1;
-    UINT16    PmeInterrupt                  : 1;
-    UINT16    CrsSoftwareVisibility         : 1;
-    UINT16    Reserved                      : 11;
+    UINT16    SystemErrorOnCorrectableError      : 1;
+    UINT16    SystemErrorOnNonFatalError         : 1;
+    UINT16    SystemErrorOnFatalError            : 1;
+    UINT16    PmeInterrupt                       : 1;
+    UINT16    ConfigurationRrsSoftwareVisibility : 1;
+    UINT16    NoNfmSubtree                       : 1;
+    UINT16    Reserved                           : 10;
   } Bits;
   UINT16    Uint16;
 } PCI_REG_PCIE_ROOT_CONTROL;
 
 typedef union {
   struct {
-    UINT16    CrsSoftwareVisibility : 1;

```

Comment:
Why change this field name? What impact caused by this change?

Reply:
@lgao4 The change was done to sync with the updated specification terminology. But prioritizing compatibility this has been reverted.